Transparency on Titlepage

EagleoutIce opened the pull request:
As requested, i have added a toggle to allow for transparency on the titlepage (although it is rather hard to see with the sample picture):

![Screenshot_20240404_184313](https://github.com/SEatUPB/FancyBeamer/assets/9303573/9f933c53-0314-49a0-a02e-dbc61e4bf0c8)

Currently, the toggle is rather simple: a documentclass option `transparenttitle`. So, to open the discussion:

* do you want the transparency itself to be configurable? (currently it is fixed to `0.72`, which i quite like). 
* what name should the option/macro to enable the transparency have? something like `\settitlefootertransparency{.5}` feels... long

tthuem commented:
The default value for the transparency sounds reasonable. Allowing it to be changed by users seems useful and the proposed name sounds just fine for me. As this is only used once, I have not problems with a longer macro name.

EagleoutIce commented:
ok, so to unify the api i have:
* removed the newly introduced and inflexible documentclass option 'transparenttitle' (i.e., no new option introduced by this pr)
* added the aforementioned macro `\settitlefootertransparency` which can be used to get the transparent title (e.g., like `\settitlefootertransparency{0.72}` or `\settitlefootertransparency{0.1}`)
* fixed a bug which caused the logos to be transparent as well, now they retain their intended look.

Here the much clearer example with `\settitlefootertransparency{0.3}`: 

![Screenshot_20240613_211753](https://github.com/SEatUPB/FancyBeamer/assets/9303573/5cbfb856-b63d-48b1-8d99-e9f7d63b15ed)

tthuem commented:
The changes in code look fine to me, but we probably need to document this in the readme and demo slides, too.

EagleoutIce commented:
Definitely, I may have time for that later today

tthuem commented:
@Benno2000 I thought we have talked about a readme. Does it exist already or do we have to still create one based on the old template?

EagleoutIce commented:
Should I create it? Or was there an agreement outside of this issue?

tthuem commented:
@EagleoutIce Feel free to create one if you have time for that.

EagleoutIce commented:
To show you what the docs for this feature may look like, i have merged [PR#30](https://github.com/SEatUPB/FancyBeamer/pull/30) into this one already, so we should merge this one afterwards. For specific doc updates see https://github.com/SEatUPB/FancyBeamer/pull/22/commits/edd5a0ce123d867194652d83cbe42cf05a673c3c specifically
